base/sysinfo.jl

```diff
@@ -162,7 +162,7 @@ total_memory() = ccall(:uv_get_total_memory, UInt64, ())
 Get the process title. On some systems, will always return empty string. (not exported)
 """
 function get_process_title()
-    buf = zeros(UInt8, 512)
+    buf = Vector{UInt8}(512)
```

Comment:
How do you  resize it to only print the string if using `StringVector`? Can't seem to find any documentation on `StringVector`, but doing `StringVector(512)`   and `return String(buf)` seems to work except it prints all the `\0`